zenodo/modules/pages/templates/zenodo_pages/contact_form.html

```diff
@@ -24,63 +24,206 @@
 
 {% extends "zenodo_theme/page.html" %}
 
-{% from "invenio_userprofiles/settings/_macros.html" import render_field %}
-
 {%- block page_body %}
 <div class="container">
   <div class="row">
-    <div class="col-md-3"></div>
-    <div class="col-md-6">
-      <h2>Get help with Zenodo</h2>
+    <div class="col-md-1 col-xs-0"></div>
+    <div class="col-md-10 col-xs-12">
+      <h2 class="header-form">Get help with Zenodo</h2>
       <hr />
       <div class="panel-body">
-      <form action="" method="POST" enctype="multipart/form-data">
-        {% if not current_user.is_authenticated %}
-          It is recommended to login first.
-        {% endif %}
-
-        {{ form.csrf_token }}
-        {{ render_field(form.name) }}
-
-        {% if current_user.is_authenticated %}
-          Zenodo staff will get back to you by your Zenodo email: <b>{{ form.email.data }}</b>
-        {% else %}
-          {{ render_field(form.email) }}
-        {% endif %}
-
-        {{ render_field(form.subject) }}
-        {{ render_field(form.issue_category) }}
-        <p id="issue_text"></p>
-        {{ render_field(form.description) }}
-        {{ render_field(form.attachments) }}
-
-        {{ form.include_os_browser }} Send the browser and system information.
-        <br />
-        To better address your problem,
-        the following information can be sent to us (recommended) alongside with your message:
-        <br />
-        <b>Operating system:</b> {{ form.os.data }}
-        <br />
-        <b>Browser:</b> {{ form.browser.data }}
-        <br />
-        <b>Version:</b> {{ form.version.data }}
-        <br />
-
-        {{ render_field(form.recaptcha) }}
-
-        <button class="btn btn-block btn-success" type="submit">
-          Send Request
-        </button>
-      </form>
+        <form action="" method="POST" enctype="multipart/form-data">
+          {% if not current_user.is_authenticated %}
+          <div class="alert alert-info login-email" role="alert">
+            It is recommended to <b>login</b> first.
+          </div>
+          {% else %}
+          <div class="alert alert-info login-email" role="alert">
+            Zenodo staff will get back to you by your Zenodo email:
+            <b>{{ form.email.data }}</b>.
+          </div>
+          {% endif %}
+
+          {{ form.csrf_token }}
+
+          {%- for each in form.field_sets %}
```

Comment:
`field` is a probably more descriptive name than `each` in the context of this loop